commit fed92d2986ff9180ce8069406d375b5873d41df0 (refs/changes/46/5346/6)
Author: Dan McDonald <danmcd@joyent.com>
Date:   2019-01-13T19:56:06-05:00 (9 months ago)
    
    OS-7504 varpd svp_remote_resolved exits on unowned mutex
    Reviewed by: Jason King <jbk@joyent.com>
    Reviewed by: Mike Zeller <mike.zeller@joyent.com>

diff --git a/usr/src/lib/varpd/svp/common/libvarpd_svp_conn.c b/usr/src/lib/varpd/svp/common/libvarpd_svp_conn.c
index 5d19f8a388..4d10d1dba4 100644
--- a/usr/src/lib/varpd/svp/common/libvarpd_svp_conn.c
+++ b/usr/src/lib/varpd/svp/common/libvarpd_svp_conn.c
@@ -919,7 +919,6 @@ svp_conn_fallout(svp_conn_t *scp)
 		    "unkonwn state");
 	}
 	mutex_exit(&scp->sc_lock);
-	mutex_exit(&srp->sr_lock);
 }
 
 int
